THIRD-PARTY-LICENSES.csv

```diff
@@ -805,6 +805,7 @@
 "make-fetch-happen@10.2.1","ISC","https://github.com/npm/make-fetch-happen"
 "make-fetch-happen@5.0.2","ISC","https://github.com/zkat/make-fetch-happen"
 "marked@4.2.3","MIT","https://github.com/markedjs/marked"
+"mat-select-search@1.2.21","Custom: https://github.com/shafi-sahal/MatSelectSearch",""
```

Comment:
[Was a big thread ](https://github.com/angular/components/issues/5697), didn't fully read through. 
Did you consider https://github.com/bithost-gmbh/ngx-mat-select-search which seems to be more popular?
(this is a discussion, I know little about the two aside from seeing both in the thread and this one seems to have more commits/stars)

Reply:
I tried ngx-mat-select-search but I couldn't get it to compile due to various module errors like `ERROR in ./node_modules/ngx-mat-select-search/fesm2020/ngx-mat-select-search.mjs 842:108-124 export 'MatIconButton' (imported as 'i4') was not found in '@angular/material/button' (possible exports: MatAnchor, MatButton, MatButtonModule)`

Reply:
Did you try version 5? 
https://github.com/bithost-gmbh/ngx-mat-select-search#compatibility

Reply:
no, i'll try that out right now!

Reply:
changed to ngx-mat-select-search as it seemed to resolve Charles' issues above